indico/web/client/js/react/components/pictures/PictureManager.jsx

```diff
@@ -0,0 +1,406 @@
+      const img = await loadImage(imageSrc);
+      if (Math.min(img.naturalWidth, img.naturalHeight) < minPictureSize) {
+        setCustomFileRejections({
+          code: 'min-size',
+          message: Translate.string(
+            'The picture you uploaded is {width} by {height} pixels. Please upload a picture with a minumum of {minPictureSize} pixels on its shortest side.',
+            {width: img.naturalWidth, height: img.naturalHeight, minPictureSize}
+          ),
+        });
+        rejected = true;
+      }
+    }
+    if (!rejected) {
+      setCustomFileRejections(null);
+      dispatch({type: 'START_EDITING', imageSrc});
+    }
+  };
+
+  const dropzone = useDropzone({
+    onDropAccepted,
+    disabled: disabled || (isUploading || failed || isCapturing || isEditing),
+    accept: ['.png', '.jpg', '.gif', '.webp'],
+    multiple: false,
+    noClick: true,
+    noKeyboard: true,
+  });
+
+  let picture = null;
+  if (!isInitialPicture && isEditing) {
+    picture = {
+      imageSrc: pictureState.imageSrc,
+    };
+  } else if (
+    !isInitialPicture &&
+    !isCapturing &&
+    (pictureState.picture.uuid === null || pictureState.picture.uuid === value)
+  ) {
+    picture = {
+      filename: pictureState.picture.filename,
+      size: pictureState.picture.size,
+      upload: {
+        failed,
+        ongoing: isUploading,
+        finished: uploadFinished,
+        progress: pictureState.progress,
+      },
+    };
+  } else if (initialPictureDetails !== null && value === initialPictureDetails.uuid) {
+    picture = initialPictureDetails;
+  }
+
+  let pictureAction = null;
+  if (failed || uploadFinished) {
+    pictureAction = {onClick: reset, icon: 'undo'};
+  } else if (isInitialPicture && !required) {
+    pictureAction = {onClick: clear, icon: 'x', color: 'red'};
+  }
+
+  useEffect(() => {
+    if (!setValidationError) {
+      return;
+    }
+    if (isUploading) {
+      setValidationError(Translate.string('Upload in progress'));
+    } else if (failed) {
+      setValidationError(Translate.string('Picture upload or Capture failed'));
+    } else if (isCapturing) {
+      setValidationError(Translate.string('Picture is being captured'));
+    } else if (isEditing) {
+      setValidationError(Translate.string('Picture editing in progress'));
+    } else {
+      setValidationError(undefined);
+    }
+  }, [setValidationError, isUploading, failed, isCapturing, isEditing]);
+
+  const capture = {
+    pictureCamera: Camera,
+    isCameraActive: isCapturing,
+    isCameraAllowed,
+    onOpenCameraDialog,
+  };
+
+  const cropper = {
+    pictureCropper: Cropper,
+    isEditActive: isEditing,
+  };
+
+  const descriptionText = minPictureSize
+    ? Translate.string(
+        'Minimum picture width or height: {minPictureSize} pixels. Allowed file types: (.jpg, .png, .gif, .webp)',
+        {minPictureSize}
+      )
+    : Translate.string('Allowed file types: .jpg, .png, .gif');
```

Comment:
```suggestion
    : Translate.string('Allowed file types: jpg, png, gif, webp');
```

(i guess you just forgot webp here?)